sirepo/job_supervisor.py

```diff
@@ -671,6 +684,8 @@ def _req_is_valid(self, req):
     def _status_reply(self, req):
         def res(**kwargs):
             r = PKDict(**kwargs)
+            if self.db.get('cancelledDueToTimeout'):
```

Comment:
Here's where we may want a lazy db upgrade. cancelledDueToTimeout should appear in the reply as False, always, so other code doesn't have to use get.

So we need some type of fixup that happens on ComputeJob init. We will want to add items to the history, too.

Reply:
I've added some fixup code. It is enough? Should we start versioning the db files?

https://github.com/radiasoft/sirepo/pull/2373/files#diff-940c7c8cb02a032b1e9c4befe02e6b81R312